scripts/uosc.lua

```diff
@@ -205,6 +205,7 @@ local defaults = {
 	ui_scale = 1,
 	font_scale = 1,
 	text_border = 1.2,
+	text_width_estimation=false,
```

Comment:
diff with conf `text_width_estimation=yes`

Reply:
conf files use yes/no for booleans

Reply:
I mean uosc.conf 's value doesn't match script's default value.

Reply:
Accroding to the previous commits and prs, tomasklaen always use the same value.

Reply:
Ah yes, didn't even notice. Thanks